src/azure-cli/azure/cli/command_modules/monitor/aaz/latest/monitor/autoscale/_update.py

```diff
@@ -0,0 +1,1108 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "monitor autoscale update",
+)
+class Update(AAZCommand):
+    """Update an autoscale setting.
+
+    For more information on autoscaling, visit: https://docs.microsoft.com/azure/monitoring-and-diagnostics/monitoring-understanding-autoscale-settings.
+
+    :example: Update autoscale settings to use a fixed 3 instances by default.
+        az monitor autoscale update -g {myrg} -n {autoscale-name} --count 3
```

Comment:
Please don't use placeholder characters in examples. You should use fake value in examples.
```suggestion
        az monitor autoscale update -g myrg -n autoscale-name --count 3
```